### DISCUSSION TITLE
Dashboard

### DISCUSSION THREAD

**Discussion by markusjnagel** [sonnen-tools/node-red#9] [Ideas]
During an email discussion, a user suggested to implement a simple dashboard in addition to the static status page.

It should contain (minimum viable product):

Production - Consumption - State of Charge - Grid Feed-in

plus (my addition) a short status about loading schedule and Tibber prices.

Similar to this (which is a full-fledged Grafana, so it won't be as sophisticated)
![image](https://github.com/sonnen-tools/node-red/assets/33287465/1233179b-b971-49b7-9b10-86dc6ccfe751)

The "original" Node-RED Dashboard uses outdated libraries that are no longer maintained (and pose a security risk, aside from the fact that nobody with half a brain uses them anymore for new developments) - so I started looking into this 
https://dashboard.flowfuse.com/ 

**NOTE:** We should keep this simple - anyone who needs a deep integration and super flexible dashboards should look into storing the data in InfluxDB or Prometheus and visualise via Grafana. Which is what the big boys do ;-) But since we're just playing around with Visualisation, we might be better off with a simple, Node-RED built in solution (well, imported into Node-RED but not an external piece of software or two).

**Comment by doncuco** [comment]
I like the dashboard idea and your first version is already great.

Could you add diagrams label to the axis as well?
And could you add a red line or something for the actual timestamp?

in the current v.0.0.2 my dashboard looks like this after the strange reboot thing of nodered last night.

<img width="1053" alt="image" src="https://github.com/sonnen-tools/node-red/assets/122402791/40bfc243-be70-41d8-a777-3a0b0c5a5a12">

**Comment by Olli510** [comment]
Very helpful project, really. Thynk you for that.

Is there any further progression? Release of 0.04 maybe?
